libs/crypto/keypair.hpp

```diff
@@ -86,9 +86,9 @@ namespace iroha {
       }
 
       /**
-        * Build a keypair with public key in binary format
-        * @param pub
-        */
+       * Build a keypair with public key in binary format
+       * @param pub
+       */
       explicit Keypair(const uint8_t *pub) : has_private(false) {
```

Comment:
Is it OK to construct a pair with only one object? Does not seem like a pair anymore to me